### DISCUSSION TITLE
Latest version 1.3.6?

### DISCUSSION THREAD

**Discussion by djrm05** [echavet/MitsubishiCN105ESPHome#136] [General]
Hi @echavet , I was updating ESPHOME to 2024.7.0 tagging 1.3.5 and saw the compilation error due to uptime_sensor.... I have just checked your version 1.3.6  and it included the change for this uptime sensor, so tried and worked correctly. 

Could we get this 1.3.6 version as latest stable now?

Thanks!!

**Comment by echavet** [comment]
Done

**Comment by djrm05** [comment]
Great, thanks!
